doc/api/embedding.md

```diff
@@ -167,8 +167,1327 @@ int RunNodeInstance(MultiIsolatePlatform* platform,
+  `stdio` file descriptor or TTY state.
+- `node_embedding_platform_flags_no_default_signal_handling` - Do not register
+  Node.js-specific signal handlers and reset other signal handlers to
+  default state.
+- `node_embedding_platform_flags_no_init_openssl` - Do not initialize OpenSSL
+  config.
+- `node_embedding_platform_flags_no_parse_global_debug_variables` - Do not
+  initialize Node.js debugging based on environment variables.
+- `node_embedding_platform_flags_no_adjust_resource_limits` - Do not adjust OS
+  resource limits for this process.
+- `node_embedding_platform_flags_no_use_large_pages` - Do not map code segments
+  into large pages for this process.
+- `node_embedding_platform_flags_no_print_help_or_version_output` - Skip
+  printing output for `--help`, `--version`, `--v8-options`.
+- `node_embedding_platform_flags_generate_predictable_snapshot` - Initialize the
+  process for predictable snapshot generation.
+
+#### Callback types
+
+##### `node_embedding_handle_error_callback`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+typedef node_embedding_status(NAPI_CDECL* node_embedding_data_release_callback)(
+    void* data);
+```
+
+Function pointer type to release data.
+
+The callback parameters:
+
+- `[in] data`: The data to be released.
+
+##### `node_embedding_platform_configure_callback`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+typedef node_embedding_status(
+    NAPI_CDECL* node_embedding_platform_configure_callback)(
+    void* cb_data,
+    node_embedding_platform_config platform_config);
+```
+
+Function pointer type for user-provided native function that configures the
+platform.
+
+The callback parameters:
+
+- `[in] cb_data`: The data associated with the callback.
+- `[in] platform_config`: The platform configuration.
+
+#### Functions
+
+##### `node_embedding_last_error_message_get`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Gets the last error message for the current thread.
+
+```c
+const char* NAPI_CDECL
+node_embedding_last_error_message_get();
+```
+
+Returns a pointer to a non-null string that contains the last error message
+in the current thread or to empty string if there were no error messages.
+
+The value of the string should be copied or processed immediately since it can
+be overridden by upcoming embedding API calls.
+
+##### `node_embedding_last_error_message_set`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Sets the last error message for the current thread.
+
+```c
+const char* NAPI_CDECL
+node_embedding_last_error_message_set(const char* message);
+```
+
+- `[in] message`: The new last error message for the current thread.
+
+The stored value can be retrieved by the `node_embedding_last_error_message_get`
+function.
+
+The last error message can be removed by passing null to this function.
+
+##### `node_embedding_main_run`
```

Comment:
I'm not sure this function belongs in this section. It's not really a platform API but one which uses all of the apis to provide an easy way to run like the Node.js binary.